settings.gradle.kts

```
@@ -1,5 +1,6 @@
 import org.gradle.kotlin.dsl.support.serviceOf
 
+include(":common", ":wearos_app")
 include(":common", ":app")
```

Can't this just be one line?
```suggestion
include(":common", ":app", ":wearos_app")
```